hls4ml/converters/keras_to_hls.py

```diff
@@ -118,6 +118,8 @@ def parse_default_keras_layer(keras_layer, input_names):
         layer['activation'] = keras_layer['config']['activation']
     if 'epsilon' in keras_layer['config']:
         layer['epsilon'] = keras_layer['config']['epsilon']
+    if 'recurrent_activation' in keras_layer['config']:
```

Comment:
Activations are separated out here because in hls4ml we do them separately from the layer in which they are defined. Since that is not the case for RNNs, `recurrent_activation` should not be parsed here, but rather in the appropriate handler.